src/main/java/com/hubspot/jinjava/lib/expression/DefaultExpressionStrategy.java

```diff
@@ -41,6 +43,14 @@ public RenderedOutputNode interpretOutput(
     if (interpreter.getContext().isAutoEscape() && !(var instanceof SafeString)) {
       result = EscapeFilter.escapeHtmlEntities(result);
     }
+    final FeatureActivationStrategy feat = interpreter
+          .getConfig()
+          .getFeatures()
+          .getActivationStrategy(ECHO_UNDEFINED);
+
+  if (result.isEmpty() && feat.isActive(interpreter.getContext())){
+      result = master.getImage();
+  }
```

Comment:
If the expression renders an empty string, `{{ '' }}` then we should allow an empty string to be output.

If instead we moved this to line 24:
```java
if (var == null && feat.isActive(interpreter.getContext()) {
  return new RenderedOutputNode(master.getImage());
}
```
We can make sure that we're only echoing back when it's undefined.

Also, make sure that you run `mvn prettier:write` to format the code, as we have tab spacing set to `2` for this project.